network/phonebook.go

```diff
@@ -175,6 +182,12 @@ func (mp *MultiPhonebook) GetAddresses(n int) []string {
 	return out
 }
 
+func (mp *MultiPhonebook) addAddressArrayToAdressSet(addrMap *map[string]bool, addrArray *[]string) {
```

Comment:
doesn't actually need to be a member of MultiPhonebook since it never uses the `mp` reference. generically it might be `stringMapSetUpdate(...)`